benchmarks/src/main/java/io/grpc/benchmarks/Utils.java

```diff
@@ -220,6 +224,38 @@ public static ManagedChannel newClientChannel(Transport transport, SocketAddress
     }
     if (directExecutor) {
       builder.directExecutor();
+    } else {
+      /*
+       * When running the higher QPS (>10000) benchmarks, there can sometimes be bad behavior using
+       * the default channel executor.  To address this, we use a fixed sized thread pool, which is
+       * specially made to not have contention on the internal queue of Runnables.  This allows for
```

Comment:
I know, but this sentence is explaining why the default executor is not appropriate, which is a cached thread pool.